flocker/node/agents/test/test_cinder.py

```diff
@@ -61,3 +65,59 @@ def test_verify_false_ca_path(self):
             'verify_ca_path': '/a/path'
         }
         self.assertEqual(_openstack_verify_from_config(**config), False)
+
+
+class FakeCinderClient(object):
+    """"
+    Mock implementation of the cinder volume manager to use in the Cinder
+    Destroy tests.
+    This class will actually do nothing as we want to emulate a non-responsive
+    cinder volume.
+    """
+    def delete(self, volume_id):
+        """
+        A no-op delete.
```

Comment:
Minor nit: typically we'd have extra line break between first sentence and rest of docstring.